analytics/lib/counts.py

```diff
@@ -873,6 +885,21 @@ def get_count_stats(realm: Optional[Realm] = None) -> Dict[str, CountStat]:
             )
         )
 
+        count_stats_.append(
+            LoggingCountStat(
+                "mobile_pushes_received_r::day",
+                RemoteRealmCount,
+                CountStat.DAY,
+            )
+        )
+        count_stats_.append(
+            LoggingCountStat(
+                "mobile_pushes_forwarded_r::day",
+                RemoteRealmCount,
+                CountStat.DAY,
+            )
+        )
```

Comment:
These cannot be `"mobile_pushes_received::day"`, because the names is taken by the `RemoteInstallationCount`s above and they can't overlap. Was going to call this `"mobile_pushes_received_realm::day", but that hits our varchar(32) limit... Any ideas? :neutral_face:

Reply:
Possibly the best fix is to just to change that to a `varchar(40`)?

Reply:
OK, thinking about this more, I think that'd be the wrong solution; these two are fundamentally the same CountStat, and the thing that varies is just whether the self-hosted server has an identifiable `RemoteRealm`.

So in some sense, what we want to be doing, is this:
```
     do_increment_logging_stat(
         remote_realm || server,
         COUNT_STATS["mobile_pushes_forwarded::day"],
         None,
         timezone_now(),
         increment=android_successfully_delivered + apple_successfully_delivered,
     )
```

But that doesn't actually work. I think we should give the good name to the `Realm`-level one, since that's the main long-term feature. And we can fit `mobile_pushes_forwarded_old::day`, so let me try that.

Reply:
Hmm, actually, `do_aggregate_to_summary_table` does not do any `RemoteInstallationCount` aggregation; and if we added that, we could exclude these tables from it. So an alternative option would be to just have the names collide and have documentation for why it's that way.

Reply:
Pushed a version with a new proposal for how to handle this detail.